src/daft-scan/src/lib.rs

```diff
@@ -1003,6 +1004,45 @@ mod test {
         )
     }
 
+    fn make_glob_scan_operator(num_sources: usize) -> GlobScanOperator {
+        let file_format_config: FileFormatConfig = FileFormatConfig::Parquet(ParquetSourceConfig {
+            coerce_int96_timestamp_unit: TimeUnit::Seconds,
+            field_id_mapping: None,
+            row_groups: None,
+            chunk_size: None,
+        });
+
+        let mut sources: Vec<String> = Vec::new();
+
+        for _ in 0..num_sources {
+            sources.push(format!("../../tests/assets/parquet-data/mvp.parquet"));
+            // sources.push(format!("File {}", i + 1));
```

Comment:
commented out line.

Reply:
removed the commented line.

On a side note, I am not sure of the failing benchmarking tests. The numbers also seem highly varying, though there was no change in functionality.

Reply:
@anmolsingh20 the benchmarks are a known issue right now, you can just ignore them for the time being.